### ISSUE TITLE
install instructions to test the snap appear before the snap is in the store

### ISSUE THREAD

**Issue by elopio** [canonical-ols/build.snapcraft.io#718]
I'm building a new snap, and it shows the text:

> To test this snap on your PC or cloud instance:
> 
> sudo snap install --edge dillinger-elopio 

The problem is that the snap is not yet in the store, so if I run those commands they will fail. This text should be visible only after the snap has been built and published to edge.

![selection_004](https://cloud.githubusercontent.com/assets/617831/25729822/2d5a6004-30f5-11e7-9b79-bb3b9aff6bca.png)

**Comment by evandandrea** [comment]
Duplicate of #528

**Comment by elopio** [comment]
Oh, already fixed. Magic! thanks.

**Comment by evandandrea** [comment]
On staging 😄
